Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -436,6 +436,54 @@ Next, navigate to a new website in Safari. You should see traced function calls
  21324 ms     | -[NSURLRequest initWithURL:0x106388b00 cachePolicy:0x0 timeoutInterval:0x106388b80
 ```
 
+### Binary Analysis
+
+An introduction to binary analysis using binary analysis frameworks has already been discussed in the "[Dynamic Analysis](0x05c-reverse-engineering-and-tampering#dynamic-analysis "Dynamic analysis")" section for Android. We recommend you to revisit this section and refresh the concepts on this subject.
+
+For Android, we used Angr's symbolic execution engine to solve a challenge. In this section, we will revisit the Angr binary analysis framework to analyze the [UnCrackable Level 1 crackme app](https://github.com/OWASP/owasp-mstg/blob/master/Crackmes/iOS/Level_01/UnCrackable_Level1.ipa "UnCrackable Level 1 iOS App") but instead of symbolic execution we will use its concrete execution (or dynamic execution) features.
+
+#### Angr
+
+Angr is a very versatile tool, providing multiple techniques to facilitate binary analysis, while supporting various file formats and hardware instructions sets.
+
+> The Mach-O backend in Angr is not well-supported, but it works perfectly fine for our case.
+
+While manual analyzing the code in the [Reviewing Disassembled Native Code](#reviewing-disassembled-native-code "reviewing disassembled native code")" section, we reached a point where performing further manual analysis was cumbersome. The function at offset `0x1000080d4` was identified as the final target which contains the secret string.
+
+If we revisit that function, we can see it involves multiple sub-function calls and interestingly none of these functions have dependency on other library calls or system calls. In a nutshell, these functions are self contained and can be executed without needing any other platform specific libraries. This is a perfect case to use Angr's concrete execution engine. Follow the steps below to solve this challenge using Angr:
+
+- Get the ARM64 version of the binary using `lipo` tool (ARMv7 can be used as well).
```

Comment:
As it is the name of the tool we don't use backticks, if you refer to the command it would include backticks (e.g. run `lipo <binary>` to ...).
```suggestion
- Get the ARM64 version of the binary using lipo tool (ARMv7 can be used as well).
```